### PR TITLE
Add the CSS in the new detail page

### PR DESCRIPTION
**Pull request by rememberlenny**
# Content

I ported over the scss files used in the new details page.

I stripped out as much unnecessary boilerplate files.

## Note

We are using two CSS frameworks here, but the new one is only used on the new pages. Eventually, we will be entirely phasing out the Bootstrap CSS from the current design.

Also, these dont match up with designs perfectly. They are a port of the existing styles.

## Trello 

https://trello.com/c/DB40MGLi/397-transition-styles-design-new-details-branch-into-master

### CODE REVIEW

**Comment by nickbristow** [comment]
Thats a lot of files, but it looks ok to me.

File: app/assets/stylesheets/redesign/_global.scss [hunk 1]
```diff
@@ -0,0 +1,637 @@
+// Foundation for Sites by ZURB
```
Comment (jessieay) [inline app/assets/stylesheets/redesign/_global.scss]: should this be in `vendor/assets` since it came from an outside lib?

**Comment by jessieay** [comment]
can we move any scss we didn't write into `vendor` ? http://guides.rubyonrails.org/asset_pipeline.html#asset-organization

That will also help with code review bc we don't need to look closely at vendored css

File: app/assets/stylesheets/redesign/_header.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+.container {
+    margin-right: auto;
+    margin-left: auto;
+    padding-left: 15px;
+    padding-right: 15px;
+}
+
+/* line 4, /Users/leonardbogdonoff/Spaceship/rails/c2/app/assets/stylesheets/common/header.scss */
```
Comment (yozlet) [inline app/assets/stylesheets/redesign/_header.scss]: Where are these comments coming from? Some build process?

Comment (rememberlenny) [inline app/assets/stylesheets/redesign/_header.scss]: Not sure, but removed @yozlet

**Comment by rememberlenny** [comment]
Updated the CSS and moved it all to `vendor` @jessieay @nickbristow

**Comment by rememberlenny** [comment]
Tests passing

**Comment by pkarman** [comment]
@rememberlenny methinks @jessieay meant the top-level `vendor/` directory, not creating a new vendor in `app/assets/`.

**Comment by rememberlenny** [comment]
@pkarman Got it!

**Comment by rememberlenny** [comment]
@pkarman Moved files!

**Comment by pkarman** [comment]
looks good. happy to merge once tests pass.

**Pull request merged**